src/drm.hpp

```diff
@@ -171,7 +171,16 @@ enum drm_mode_generation {
 	DRM_MODE_GENERATE_FIXED,
 };
 
+enum drm_mode_rotate {
```

Comment:
Can we avoid using the `drm_mode_` namespace for this new enum? It's supposed to be reserved for libdrm.

Also I'd prefer to use 90/180/270 degrees instead of right/left/etc.

Reply:
OK. Will try to come up with another name.
Should the options for rotation still be `left` `right` or we should change that to '90', `180` etc?

Reply:
Name changed to g_panel_orientation for the enum. The options are still "left", "right", "normal", "upsidedown". I think they are more intuitive for command line.